GraphShow.py

```diff
@@ -88,7 +88,7 @@ def create_page(self, events):
 
```

Comment:
这一变更是合理的，确保了文件写入时正确处理 UTF-8 字符，提升了代码的可靠性和兼容性。

```suggestion
无进一步建议，变更符合评审标准。